lib/Backends/OpenCL/kernels.cl

```diff
@@ -6,20 +6,26 @@ typedef unsigned cl_uint32_t;
 /// This type is always 64 bits.
 typedef unsigned long cl_uint64_t;
 
+#if GLOW_HOST_SIZEOF_SIZE_T == 4
```

Comment:
This is not correct. The host does not have to match the device.

Reply:
For quick fix the inconsistence between Host and Device.